src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    const keys = robot[key];
+    const values = key;
```

Comment:
same here? why plural?
```suggestion
    const value = key;
```